src-ghc/Pact/ApiReq.hs

```diff
@@ -51,12 +51,25 @@ import Pact.Types.Runtime hiding (PublicKey)
 import Pact.Types.API
 
 
-data KeyPair = KeyPair {
+data ApiKeyPair = ApiKeyPair {
   _kpSecret :: PrivateKey,
-  _kpPublic :: PublicKey
+  _kpPublic :: PublicKey,
+  _kpScheme :: Maybe AddressFormat
```

Comment:
Go back to PPKScheme type here

Reply:
Keep `ED25519` for current scheme

Reply:
ED25519 = Blake512B + ED25519 + Full addys
CW = Blake256B + ED25519 + Blake128B addys?
ETH = ... + secp256k1 + eth addys